example/data/collections/syncGranule/duplicate_handling_error.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "MOD09GQ_duplicateHandlingError",
+  "version": "006",
+  "dataType": "MOD09GQ",
```

Comment:
The collection json with name <dataType>___<version> (e.g. MOD09GQ___006.json) will be stored in s3,  and can overwrite existing file generated from setup.  So dataType should be different as well.

Reply:
OK, I'll update that.